doc/plugins/proxy-rewrite-cn.md

```diff
@@ -0,0 +1,73 @@
+[English](proxy-rewrite.md)
+# proxy-rewrite
+
+上游代理信息重写插件。
+
+#### 配置参数
+|名字    |可选|说明|
+|-------         |-----|------|
+|scheme          |可选| 转发到上游的新`schema` 协议，可以是`http`或`https`，默认`http`协议|
+|uri             |可选| 转发到上游的新`uri` 地址|
+|host            |可选| 转发到上游的新`host` 地址，格式可以为`192.168.80.128:8080`或`192.168.80.128`如果未设置端口将默认设置为`80`，此配置优先级将高于`upstream.nodes` |
+|enable_websocket|可选| 是否启用`websocket`（布尔值），默认不启用|
+
+### 示例
+
+#### 开启插件
+下面是一个示例，在指定的 route 上开启了 proxy rewrite 插件:
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+{
+    "methods": ["GET"],
+    "uri": "/test/index.html",
+    "plugins": {
+        "proxy-rewrite": {
+            "uri": "/test/home.html",
+            "scheme": "http",
+            "host": "192.168.80.128:8080",
+            "enable_websocket": true
+        }
+    },
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "127.0.0.1:80": 1
+        }
+    }
+}'
+```
+
+#### 测试插件
+基于上述配置进行测试，测试前在 Nginx 配置文件 access 日志输出配置中补充 `$upstream_scheme $upstream_addr $upstream_uri` 变量。
```

Comment:
APISIX not support to change the format of access log, so this method is not good enough. 
I think check the asscees log of upstream server is better.